build/package.yml

```diff
@@ -60,7 +60,14 @@ steps:
   - task: PublishBuildArtifacts@1
     displayName: 'publish corstestconfiguration.json'
     inputs:
-      pathToPublish: './corstestconfiguration.json'
+      pathToPublish: './test/configuration/corstestconfiguration.json'
```

Comment:
This can now be combined with the other `PublishBuidlArtifact` step below.

Reply:
Can probably use * to copy all files under the folder now right?

Reply:
I think you may be able to just give it a folder and it will take everything.

Reply:
I initially went the * route and the build failed ([commit]( https://github.com/Microsoft/fhir-server/pull/382/commits/0b67c56b2ba82d30861d4ab5e1e4a5a0b6af4477)). Then I realized that wildcard characters are not supported ([reference](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-build-artifacts?view=azure-devops)).

I was hesitant to include the whole folder because I thought it would be better to be specific. I'll update it to include the whole configuration folder.